examples/apollo/cosmos.proxies.js

```diff
@@ -1,20 +1,10 @@
 import createApolloProxy from 'react-cosmos-apollo-proxy';
 
-const typeDefs = `
-  type Query {
-    hello(who: String): String
-  }
-`;
-
-const mocks = {
-  Query: () => ({
-    hello: (root, { who }) => `Hello ${who ? who : 'C O S M O S'}`
-  })
-};
+// see more examples at:
+// https://github.com/react-cosmos/react-cosmos#react-apollo-graphql
 
 export default [
   createApolloProxy({
-    typeDefs,
-    mocks
+    endpoint: 'https://1jzxrj179.lp.gql.zone/graphql'
```

Comment:
Definitely!

Reply:
> Hmm, doing a live call in tests can lead to lots of pain down the road, because all of a sudden the test suit doesn't down offline nor if gql.zone is down.

You are the voice of the reason, excellent point. I agree with your approach.

I replicate the `fetch-proxy` behavior in the tests (they fetch `http://xyz` and no real call is made when running the tests).

In order to give the user a feeling of the real behavior, I'm going to download the live API to create a local GraphQL API palce in a `.local-graphql-api` directory, inside the example.

☺️ This ensures pure encapsulation for the tests as well as high-fidelity example